bpf/bpf_lxc.c

```diff
@@ -470,8 +470,7 @@ static __always_inline int handle_ipv6_from_lxc(struct __ctx_buff *ctx, __u32 *d
 	return CTX_ACT_OK;
 }
 
-declare_tailcall_if(__or3(is_defined(ENABLE_IPV4), is_defined(ENABLE_IPV6),
-			  is_defined(DEBUG)), CILIUM_CALL_IPV6_FROM_LXC)
+__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV6_FROM_LXC)
```

Comment:
FWIW, I believe the original intention of this was to maintain the code structure of the previous version when we checked for an OR instead of an AND. But, maybe it does not make a lot of sense to keep it.